docs/kms/vault-config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "api_addr": "http://127.0.0.1:8200",
+  "backend": {
+    "file": {
+      "path": "vault/file"
+    }
+  },
+  "default_lease_ttl": "168h",
+  "max_lease_ttl": "720h",
+  "listener": {
+    "tcp": {
+      "address": "0.0.0.0:8200",
+      "tls_disable": 1
```

Comment:
Same here

Reply:
How you want to add that it's just a work-around this is a JSON file without comments.

Reply:
Done

Reply:
Oh yes, true..